integrated_channels/sap_success_factors/models.py

```diff
@@ -200,7 +201,15 @@ def unlink_inactive_learners(self):
         Unlink inactive SAP learners form their related enterprises
         """
         sap_learner_manager = self.get_learner_manger()
-        sap_learner_manager.unlink_learners()
+        try:
+            sap_learner_manager.unlink_learners()
+        except ClientError as exc:
+            LOGGER.error(
```

Comment:
nit: use `LOGGER.exception(` here.
also, i am trying to understand the rationale of not raising the exception again here after logging it? is this point in the code the top of the call stack?

Reply:
@brittneyexline 
this method is getting [called](https://github.com/edx/edx-enterprise/blob/master/integrated_channels/integrated_channel/tasks.py#L145) from a task `unlink_inactive_learners` which is actually on top of the call stack in the code. if we see in the code, called method is not supposed to return anything and the task itself is not doing much after calling this method.
So in the case of client error task will print error stack trace and exit.